lib/Actions.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
-var debug = require('debug')('Actions'),
-    vow = require('vow'),
-    _ = require('lodash'),
-    Storage = require('./Storage');
+const debug = require('debug')('Actions');
+const vow = require('vow');
+const _ = require('lodash');
+const Storage = require('./Storage');
+const actionsFactory = require('./actions/actionsFactory');
```

Comment:
using actionsFactory to instantiate actions now, Actions.js is reduced by 300+ lines